src/logger.ts

```diff
@@ -61,5 +61,5 @@ export const logger = new Logger({
   level: toBunyanLogLevel(process.env.LOG_LEVEL || 'info'),
   name: 'probot',
   serializers,
-  stream: new bunyanFormat({ outputMode: toBunyanFormat(process.env.LOG_FORMAT || 'short'), color: supportsColor.stdout })
+  stream: new bunyanFormat({ outputMode: toBunyanFormat(process.env.LOG_FORMAT || 'short'), color: supportsColor.stdout, levelInString: !!process.env.LOG_LEVEL_IN_STRING })
```

Comment:
where does `LOG_LEVEL_IN_STRING` come from? Would it be a new environment variable? If so, it needs to be documented. Can you elaborate on why you need this setting?

Reply:
The current logs print `{"level": 20, ...}`, by setting `levelInString` the log line will look like `{"level": "debug", ...}` This helps to print the log level as a string instead of int, which is useful for tools that process logs. 

I used `process.env.LOG_LEVEL_IN_STRING` following the same pattern used for the log format and the log level. Let folks set it for their `probot` process but don't change the current behavior.

Reply:
Wouldn't it maybe better to set "level" as string by default? It would be a breaking change, unless there is a legit reason for people wanting the number, we can as well just set `levelInString` to true.

I'm reluctant adding more configuration options, I'd rather get rid of some :)

Reply:
I'm 100% with you with not adding a new config option. The reason I went with the option is that this may be considered a breaking change. If someone is using a `json` long and it's processing the log with some tool (`filebeat`, or any tool) and expects level to be an int, this is going to break their scenarios.

If you're confident it's ok to default to true, then I'm all for it.

Reply:
No, I mean I would release it as part of Probot 10.

We could add the new option but have to remember to remove it again :D Can you add documentation for the new option?

Reply:
done, thanks for the review :)